src/jsglue.cpp

```diff
@@ -710,6 +712,8 @@ static size_t MallocSizeOf(const void* aPtr)
     return malloc_size((void*)aPtr);
 #elif defined(__MINGW32__) || defined(__MINGW64__)
     return _msize((void*)aPtr);
+#elif defined(_MSC_VER)
+    return _msize((void*)aPtr);
```

Comment:
Can you consolidate the mingw/msvc blocks in the two changes you made to this file?